### PR TITLE
Add support for imx8mmsolidrun (SD card support)

### PR DESCRIPTION
**Pull request by kivaisan**
Add support for Hummingboard Solid with booting from SD card.

Boot mode configuration: https://developer.solid-run.com/knowledge-base/hummingboard-pulse-ripple-boot-select/

Ethernet cable must be connected to CON1 -connector (closest to power connector).

For eMMC flashing device must be connected to PC from the top most USB port with USB type-A to USB type-A OTG cable (e.g. https://www.amazon.com/gp/product/B01KRO8D20).

### CODE REVIEW

**Comment by kivaisan** [comment]
I had accidentally removed imx8mmevk boot params. Pushed a fixed for that so hopefully tests will now pass.

File: meta-imx8mmsolidrun-support/conf/machine/imx8mmsolidrun.conf [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#@TYPE: Machine
+#@NAME: SolidRun iMX8MM based platforms
+#@SOC: i.MX8MM
+#@DESCRIPTION: Machine configuration for SolidRun iMX8MM based platforms
+#@MAINTAINER: Ricardo Salveti <ricardo@foundries.io>
+
+MACHINEOVERRIDES =. "mx8:mx8m:mx8mm:imx8mmevk:"
```
Comment (JanneKiiskila) [inline meta-imx8mmsolidrun-support/conf/machine/imx8mmsolidrun.conf:7]: `imx8mmevk` should be removed?

**Comment by kivaisan** [comment]
This passed tests now so maybe this could be reviewed and merged to get into next release? I don't think inheritance from imx8mmevk is that critical issue that it blocks merging this.

File: meta-imx8mmsolidrun-support/conf/machine/include/lmp-factory-custom.inc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# LMP factory specific customizations (either replace or extend options as defined by meta-lmp)
```
Comment (esajaa) [inline meta-imx8mmsolidrun-support/conf/machine/include/lmp-factory-custom.inc:1]: Doesn't having this file lead to a situation where there are two lmp-factory-custom.inc files? One from here, and another from meta-lmp-support? Or is the idea to enable only one of them at time?

Comment (kivaisan) [reply meta-imx8mmsolidrun-support/conf/machine/include/lmp-factory-custom.inc:1]: Maybe this file should be removed. I'll do that.

**Comment by kivaisan** [comment]
@JanneKiiskila @esajaa @petedyerarm Please check now. We should get this merged for 2.4.

**Review by JanneKiiskila — APPROVED** [review]

**Pull request merged**